### PR TITLE
Added key for amar

### PR DESCRIPTION
**Pull request by amar266**

### CODE REVIEW

**Comment by amar266** [comment]
please  test this

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
please test this

**Comment by amar266** [comment]
please test this

File: hiera/data/secrets/common.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ rjil::system::accounts::local_users:
...
   ahmad: {realname: 'Ahmad Faheem',sshkeys: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC3WiZkgvjGCoXxo9rWoRxo+UXeu2Q5FV5fxXNWaWBP6eS7bMQaAFWv86kDY8mGR/3wMSLJZlMjWhH9GVgVgDz01NQqKrz+m+FWdu/WvNIb37Qcn9M1oZtNg5dqcIsXc/GHtsyWSqhN6A4FhXvVnnw8b163A+ANVO9//LNw+X3d5xuHilR+dhWFD7WMP4830w4XYkmDcCia3RFW4ayxFX4TP3jq7tk/9vGQgauWiLy3UZmE6jgkc/kmBlJz85AWCiWRf3kUP/Fm5tX8rmccNgDfknA7gIDM270wxeO+MeizIQNXfaM0j02ADWGLrFZ51Gg5ZEjS0JTmb3ppxxUI6Ke/ ahmad@Faheem-PC",password: "*"}
-  vaidy: {realname: 'Vaidyanath Manogaran',sshkeys: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKs2W/NT/NUqjTWvZCxWV1P38jlsk2CdwsAdqFbEIXMctCN2yePOJ4H/oONPHA0roXQfFTLfqaOqEG9a0VZVuSVo1ECWgiS3ACVIEU30GgKcz/ttjMznVQ9P8kiJH6ULI36pslnfXpWBbG7+V30SvdUY2BBr8qj03LiRjq9QpTz4+igygKB7Ie39DIocbZiNaP6C8v8Geq6ULbu7HS4UPPvUkmjZuytzkI3wvtgOHrmO5SI+N9mzYPmCfjF74x3d+CuVdeisu0obZaBi9Y1aaFZUchu5u0UGldzufE9LvfbdQI1SlyY+b1wPMR1tv5nBX3yUw0ag3YfIxng4FCYFIb ubuntu@ubuntu",password: "*"}
```
Comment (bodepd) [inline hiera/data/secrets/common.yaml]: did you remove someone's key? That needs to be added back

**Comment by amar266** [comment]
Corrected the mistake

**Comment by bodepd** [comment]
you need to rebase, a merge conflict just got merged.

**Pull request closed without merging**